src/base/emu-i386/simx86/tables.c

```diff
@@ -291,7 +291,7 @@ unsigned char InterOps[256] =
 	0x01,	// ENTER		0xc8
```

Comment:
Just wondering, why ENTER does not compile? :)

Reply:
Actually seriously level-0 ENTERs could be compiled, as some compilers generate it... I see thousands of them with DOOM actually. Level>0 ENTER looks super rare.